src/info/repo/author.rs

```diff
@@ -135,4 +135,80 @@ mod test {
 
         assert_eq!(author.to_string(), "75% John Doe 1500");
     }
+
+    #[test]
+    fn test_author_info_title() {
+        let author = Author::new(
+            "John Doe".into(),
+            "john.doe@email.com".into(),
+            1500,
+            2000,
+            true,
+        );
+
+        let author2 = Author::new(
+            "Roberto Berto".into(),
+            "bertolone2000@email.com".into(),
+            240,
+            300,
+            false,
+        );
+
+        let authors_info = AuthorsInfo {
+            info_color: DynColors::Rgb(255, 0, 0),
+            authors: vec![author.clone()],
+        };
+
+        let authors_info_2 = AuthorsInfo {
+            info_color: DynColors::Rgb(255, 0, 0),
+            authors: vec![author2, author],
+        };
+
+        assert_eq!(authors_info.title(), "Author");
+        assert_eq!(authors_info_2.title(), "Authors");
+    }
```

Comment:
If you're testing `title()`, you might want to make it 2 tests -- one like `test_author_info_singular_title` and another like `test_author_info_plural_title`.